lib/schnorr/vs_schnorr_wasm.js

```diff
@@ -481,7 +481,7 @@ export class KeyImportSession {
     inputMessage(msg) {
         const ptr0 = passArray8ToWasm0(msg, wasm.__wbindgen_malloc);
         const len0 = WASM_VECTOR_LEN;
-        const ret = wasm.keyimportsession_inputMessage(this.__wbg_ptr, ptr0, len0);
+        const ret = wasm.keygensession_inputMessage(this.__wbg_ptr, ptr0, len0);
```

Comment:
These are changes in KeyImportSession , which we don't use right now , so let's get it merged